### DISCUSSION TITLE
Unclear on what the single source of truth for feature requests should be

### DISCUSSION THREAD

**Discussion by jcklpe** [uswds/uswds#5994] [General]
One can request a new feature via github issue ticket:
![image](https://github.com/user-attachments/assets/d2526200-e985-4f36-9429-e03bf8cb0569)

Or as a Discussion "component request":
![image](https://github.com/user-attachments/assets/19567546-4aa5-418b-b1ec-78bd04a6312c)

It's unclear to me which of these is the preferred way of making requests or what their distinction is if any. If there is a difference I think some guidance copy would be good, and if there is no real distinction there should be consolidation.

**Comment by brunerae** [comment]
Hi @jcklpe !! Great point here. For now, any feature requests that are for new Components should be created as a Discussion. Feature requests that are just updates to existing Components or other features should be created as an Issue. But you raise a great point about how that distinction is unclear. I'll add a note for our team to update the help text on these.

**Reply by jcklpe** [reply]
Thank you! That makes sense.
